### PR TITLE
docs: remove "f" from default 'shortmess' value

### PR DESCRIPTION
**Pull request by gpanders**
The "f" flag was removed in f7da4722570617bd8927e7aa533fa9a608c45bba.
The value of the "f" flag is no longer listed in the 'shortmess'
description and it cannot be disabled, so having it in the default value
is pointless and confusing.

### CODE REVIEW

**Review by justinmk — APPROVED** [review]

**Comment by MariaSolOs** [comment]
The docs still mention the `f` option [here](https://github.com/neovim/neovim/blob/8195c180065a7f477b63f0eb4e5edb890adba0c5/runtime/doc/editing.txt#L824). We should remove it too.

**Comment by gpanders** [comment]
> The docs still mention the `f` option [here](https://github.com/neovim/neovim/blob/8195c180065a7f477b63f0eb4e5edb890adba0c5/runtime/doc/editing.txt#L824). We should remove it too.
> Also should [this "old" test](https://github.com/neovim/neovim/blob/8195c180065a7f477b63f0eb4e5edb890adba0c5/test/old/testdir/setup.vim#L20) be updated?

Good catch, addressed both.

File: test/old/testdir/setup.vim [hunk 1]
```diff
@@ -17,7 +17,7 @@ if exists('s:did_load')
   set nohidden nosmarttab noautoindent noautoread noruler noshowcmd
   set nohlsearch noincsearch
   set nrformats=bin,octal,hex
-  set shortmess=filnxtToOS
+  set shortmess=ilnxtToOS
```
Comment (zeertzjq) [inline test/old/testdir/setup.vim:20]: No

Comment (gpanders) [reply test/old/testdir/setup.vim:20]: Care to elaborate (preferably with more than one syllable)? It does not cause any tests to fail.

Comment (zeertzjq) [reply test/old/testdir/setup.vim:20]: As a check that having "f" in 'shortmess' doesn't cause an error

Comment (gpanders) [reply test/old/testdir/setup.vim]: Makes sense. Reverted.

**Review by zeertzjq — APPROVED** [review]

**Pull request merged**